src/main/java/io/lumigo/models/Span.java

```diff
@@ -49,6 +51,7 @@ public static class Info {
         private String stage;
         private String messageId;
         private List<String> messageIds;
+        private List<Map<String, String>> tags;
```

Comment:
The execution tags should be a different span from type `EnrichmentSpan` (this span is supported in our BE) you can take a look on the node / python tracers